#! /bin/sh /usr/share/dpatch/dpatch-run
## 25_mysys__default.c.dpatch by  <ch@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: mysys__default.c

@DPATCH@

--- old/mysys/default.c	2005-05-26 15:26:56.000000000 +0200
+++ new/mysys/default.c	2005-06-08 19:54:26.434959920 +0200
@@ -866,7 +866,7 @@
   if ((env= getenv("ETC")))
     *ptr++= env;
 #endif
-  *ptr++= "/etc/";
+  *ptr++= "/etc/mysql/";
 #endif
   if ((env= getenv(STRINGIFY_ARG(DEFAULT_HOME_ENV))))
     *ptr++= env;
